R/plot.R

```diff
@@ -14,3 +14,40 @@ plot.baggr <- function(x, ...) {
   l[["bg"]] <- x
   do.call(baggr_plot, l)
 }
+
+#' Set, get, and replace themes for baggr plots
+#' @rdname baggr_theme_set
+#' @importFrom ggplot2 theme_grey
+#' @importFrom bayesplot theme_default
+#' @param new New theme to use for all baggr plots
+#' @description These functions get, set, and modify the ggplot2 themes
+#' of the baggr plots. By default, these match the bayesplot's package
```

Comment:
Should link to related documentation (in this case `bayesplot`, `bayesplot::bayesplot_theme_get` or `ggplot2::theme_get`), both via inline links where other pkgs are namechecked and in See Also.

Reply:
Added this as well.